daemon/daemon_unix.go

```diff
@@ -472,28 +472,28 @@ func verifyContainerResources(resources *containertypes.Resources, sysInfo *sysi
 func (daemon *Daemon) getCgroupDriver() string {
 	cgroupDriver := cgroupFsDriver
 
-	// No other cgroup drivers are supported at the moment. Warn the
-	// user if they tried to set one other than cgroupfs
-	for _, option := range daemon.configStore.ExecOptions {
+	if daemon.usingSystemd() {
+		cgroupDriver = cgroupSystemdDriver
```

Comment:
Maybe we should warn if the passed driver name is not available instead of silently reverting to cgroupsFsDriver?